tests/playbooks/kubevirt_vm.yml

```diff
@@ -33,13 +38,15 @@
         state: "{{ item.state }}"
         ephemeral: yes
         name: "{{ item.name }}"
+        wait: no
       with_items:
         - { name: 'eph1', state: 'present' }
         - { name: 'eph1', state: 'running' }
         - { name: 'eph2', state: 'present' }
         - { name: 'eph2', state: 'running' }
       register: result
-      failed_when: result.changed != false or result.kubevirt_vm.status.phase != 'Running'
+      failed_when: result.kubevirt_vm.status.phase != 'Running'
+#     failed_when: result.changed != false or result.kubevirt_vm.status.phase != 'Running'
```

Comment:
why do we need this commented line?

Reply:
There's a 4 line explanation of what this is for at the start of the playbook.

I did it this way on purpose. If I'm making a temporary workaround which I want to get removed as soon as possible, then I want to very clearly mark all the places in the code where the workaround applies. The only chance for someone to update it in a few months so that full tests are utilized is to make it an eye sore and hard to not notice should anyone look at this file.

You know how this stuff works. If I hide these lines in git, then no one will ever bother to try to fish out the originals and figure out what to do with them, so the "workaround" will just become the permanent tests until the heat death of the universe. But if the originals are still there in plain sight, there's a chance…